retroshare-android-service/src/service.cpp

```diff
@@ -39,11 +39,19 @@ int main(int argc, char *argv[])
 #endif
 
 	QCoreApplication a(argc, argv);
+
+	RsInit::InitRsConfig();
```

Comment:
Unfortunately, without this line it won't even compile (Have I forgotten about something?).